website/docs/API.md

```diff
@@ -533,13 +533,11 @@ The `renderHook` function returns an object that has the following properties:
 
 ```jsx
 {
-  all: Array<any>
-  current: any,
-  error: Error
+  current: any
```

Comment:
Since `RenderHookResult` accepts generic `Result` parameter we should use it instead of `any` here.

Reply:
I hope my changes align with what you were thinking for this.